apps/web/src/store/use-auth-controller.ts

```diff
@@ -17,7 +17,10 @@ export function applyToken(token: string | null) {
   }
 }
 
-export function getToken(): string {
+export function getToken(location?: string): string {
+  const token = location ? new URLSearchParams(location.substring(1)).get('token') : null;
+  if (token) applyToken(token);
```

Comment:
I would transfer this to the `login.tsx` page and just check it on page load. If the token exists, calling applyToken and then redirecting to the dashboard.

Reply:
Sure thing was not sure in what exact place to add the support.